.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Prettier Check
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: 2
+  NODE: 20
+
+permissions:
+  contents: read
+
+jobs:
+  css:
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]' || github.event_name == 'workflow_dispatch'
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "${{ env.NODE }}"
+          cache: npm
+
+      - run: node --version
+      - run: npm --version
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Check Prettier
+        run: npm run prettier:check
```

Comment:
This should run with `--max-warnings=0` option to send error on warning

Reply:
When it was run with https://github.com/Orange-OpenSource/ods-charts/actions/runs/8811608553/job/24185800032?pr=214 there were only warnings and the job failed. Sounds like it's the default behavior.

Reply:
My bad, I didn't saw this job